changelog/new_support_token_auth_to_download_shared_config.md

```diff
@@ -0,0 +1 @@
+* [#9828](https://github.com/rubocop/rubocop/pull/9828): Add Token auth support to download raw yml config from private repo. ([@AirWick219][])
```

Comment:
```suggestion
* [#9827](https://github.com/rubocop/rubocop/issue/9827): Add Token auth support to download raw yml config from private repo. ([@AirWick219][])
```

Since this PR has a related issue, the changelog item should point to it.